sound/soc/sof/sof-pci-dev.c

```diff
@@ -31,17 +31,21 @@ static char *tplg_path;
 module_param(tplg_path, charp, 0444);
```

Comment:
Minor nitpicks continue. A "topology path can be used to load" ... "firmware files"?

Reply:
reworded.